queue/src/fiskaltrust.Middleware.Localization.QueueAT/RequestCommands/MonthlyClosingReceiptCommand.cs

```diff
@@ -13,25 +13,24 @@
 
 namespace fiskaltrust.Middleware.Localization.QueueAT.RequestCommands
 {
-    internal class MonthlyClosingReceiptCommand : RequestCommand
+    public class MonthlyClosingReceiptCommand : RequestCommand
     {
-        private readonly ExportService _exportService;
+        private readonly IExportService _exportService;
 
         public override string ReceiptName => "Monthly-closing receipt";
 
         public MonthlyClosingReceiptCommand(IATSSCDProvider sscdProvider, MiddlewareConfiguration middlewareConfiguration, QueueATConfiguration queueATConfiguration,
-            ExportService exportService, ILogger<RequestCommand> logger)
+            IExportService exportService, ILogger<RequestCommand> logger)
             : base(sscdProvider, middlewareConfiguration, queueATConfiguration, logger)
         {
             _exportService = exportService;
         }
 
-        public override async Task<RequestCommandResponse> ExecuteAsync(ftQueue queue, ftQueueAT queueAT, ReceiptRequest request, ftQueueItem queueItem)
+        public override async Task<RequestCommandResponse> ExecuteAsync(ftQueue queue, ftQueueAT queueAT, ReceiptRequest request, ftQueueItem queueItem, ReceiptResponse response)
         {
             ThrowIfTraining(request);
-            var response = CreateReceiptResponse(request, queueItem, queueAT, queue);
 
-            if (request.cbChargeItems?.Count() != 0 || request.cbPayItems?.Count() != 0)
+            if ((request.cbChargeItems != null && request.cbChargeItems?.Count() != 0) || (request.cbPayItems != null && request.cbPayItems?.Count() != 0))
```

Comment:
We're using this check multiple times, maybe we could introduce an extension method? 
E.g. `request.HasNoChargeAndPayItems()`